authors/apps/authentication/models.py

```diff
@@ -132,11 +136,11 @@ def get_short_name(self):
 
     def generate_jwt_token(self):
         """ Generates a token that expires in 24hrs """
-        time = datetime.now() + timedelta(hours=24)
         token = jwt.encode({
             "email": self.email,
             "username": self.username,
-            "exp": int(time.strftime('%s'))
+            "exp": datetime.utcnow()
+            + timedelta(minutes=60*3)
```

Comment:
You can simply have timedelta(hours=3)`.

Reply:
Will change that